src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipService.java

```diff
@@ -162,48 +162,48 @@ private void updateTransactionWithLinksAndPartnershipName(Transaction transactio
         transactionService.updateTransaction(transaction, loggingContext);
     }
 
-    private boolean hasExistingLimitedPartnershipSubmission(Transaction transaction) {
+    private boolean hasExistingLimitedPartnership(Transaction transaction) {
         if (transaction.getResources() != null) {
             return transaction.getResources().entrySet().stream().anyMatch(
                     resourceEntry -> FILING_KIND_LIMITED_PARTNERSHIP.equals(resourceEntry.getValue().getKind()));
         }
         return false;
     }
 
-    private void updateLimitedPartnershipSubmissionWithSelfLink(LimitedPartnershipSubmissionDao submission,
+    private void updateLimitedPartnershipWithSelfLink(LimitedPartnershipDao partnershipDao,
```

Comment:
can we rename local vars and params from partnershipDao to limitedPartnershipDao, it's confusing me and making me think it's something else.

Reply:
Can do. I based it on the PartnershipController but happy to change to prefix with limited as ultimately the data is stored in a `limited_partnerships` collection.

Reply:
I think it's better with the limited, I am finding it difficult to know what endpoint is related to what collection